server/config.js

```diff
@@ -5,7 +5,8 @@ const cache = new Map()
 
 const defaultConfig = {
   webpack: null,
-  poweredByHeader: true
+  poweredByHeader: true,
+  assetPrefix: '/'
```

Comment:
I like this. I a real CDN setup, it's possible user is not using our `buildId`. And we could simply use something like `assetPrefix` and avoid the use of buildId directly in the client side.
(Becuase buildId is in the `assetPrefix`)

Yep. Implement this.